provisions/roles/cleanup/scanner/tasks/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Remove all containers
+  sudo: yes
+  shell: docker rm -f `docker ps -a -q`
```

Comment:
@rtnpro here scanners stop by themselves, but sometimes scanner keeps containers even though scan is over. may be we can solve this in the other way round, by making sure scanner is removing all the containers and images once scan is over for some image.